int16 left_dir
int16 right_dir

